src/commodore/index.ts

```diff
@@ -87,20 +98,41 @@ export async function extractPackageFile(
   let components: CommodoreComponentDependency[];
 
   const repoDir: string | undefined = getGlobalConfig().localDir;
+  let globalDir: string = '';
+  // Tenant repos must have `commodore.tenantId` set
+  const isTenantRepo: boolean =
+    config.tenantId !== undefined && config.tenantId != '';
```

Comment:
I think this is missing a `null` check

```suggestion
  const isTenantRepo: boolean =
    config.tenantId != null && config.tenantId != '';
```

Reply:
I don't see how `tenantId` could ever be `null`, but better safe than sorry...